packages/vm/lib/evm/opcodes/EIP1283.ts

```diff
@@ -9,51 +9,49 @@ import { RunState } from './../interpreter'
  * @param {Buffer}   value
  */
 export function updateSstoreGasEIP1283(runState: RunState, found: any, value: Buffer) {
-  if (runState._common.hardfork() === 'constantinople') {
-    const original = found.original
-    const current = found.current
-    if (current.equals(value)) {
-      // If current value equals new value (this is a no-op), 200 gas is deducted.
-      runState.eei.useGas(new BN(runState._common.param('gasPrices', 'netSstoreNoopGas')))
-      return
+  const original = found.original
+  const current = found.current
```

Comment:
personal style nit ^_^
```suggestion
  const { original, current } = found
```